src/dex/solidly/config.ts

```diff
@@ -104,4 +119,7 @@ export const Adapters: Record<number, AdapterMappings> = {
   [Network.MAINNET]: {
     [SwapSide.SELL]: [{ name: 'Adapter04', index: 1 }], // solidly
   },
+  [Network.AVALANCHE]: {
+    [SwapSide.SELL]: [{ name: 'AvalancheAdapter02', index: -1 }], // TODO: solisnek
```

Comment:
Need an adapter supporting Solidly fork on Avalanche

Reply:
You can use 14 for that